src/Glpi/Form/Condition/ConditionableTrait.php

```diff
@@ -36,6 +36,8 @@
 
 use JsonException;
 
+use function Safe\json_decode;
```

Comment:
There is no unsafe usage of json_decode in this file, this doesn't seems needed.

Reply:
The lib is indeed supposed to detect the `JSON_THROW_ON_ERROR` flag, see https://github.com/thecodingmachine/phpstan-safe-rule/blob/5f9795eae8891dffa475965463a4281633651768/src/Rules/UseSafeFunctionsRule.php#L41-L64